src/modules/myposts/components/Myposts.js

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios'
+import React, { useContext } from 'react'
+import { useState } from 'react'
+import { useEffect } from 'react'
+import { UserContext } from '../../common/providers/UserContext'
+import { Post } from './Post'
+
+export const MyPosts = () => {
+    const [posts, setPosts] = useState([])
+    const { user } = useContext(UserContext)
+
+    useEffect(() => {
+        const getData = async () => {
+            try {
+                const res = await axios.get(
+                    'http://localhost:3001/post/getPost',
```

Comment:
if you merge dev in you will now see we use the url from an env file
process.env.REACT.....
Check out the code for the syntax.
And then create an env file locally for using it